### PR TITLE
FAM 1.4.0 blogpost

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: _posts/2020-10-23-foreman-ansible-modules-v140-released.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+tags:
+- foreman
+- ansible
+---
+
+Okay, you know the [drill](/2020/09/foreman-ansible-modules-v130-released.html), we have released [Foreman Ansible Modules v1.4.0](https://galaxy.ansible.com/theforeman/foreman) and here is the best-of from that release!
+
+<!--more-->
+
+The biggest non-functional change in 1.4.0 is the vendoring of the `apypie` library. The library is reposible for the communication with the Foreman API and not Ansible-specific as such. In the past it was shipped as a standalone library and users had to install it from RPM or PyPI to be able to use the collection. This turned out to be a burden, especially for users running on enterprise distributions, so we decided to vendor the library inside the collection. The library development still happens separately on [GitHub](https://github.com/Apipie/apypie). Users now only have to install the `requests` library, which is easily available on any distribution.
```
Comment (ekohl) [inline _posts/2020-10-23-foreman-ansible-modules-v140-released.md]:
```suggestion
The biggest non-functional change in 1.4.0 is the vendoring of the `apypie` library. The library is responsible for the communication with the Foreman API and not Ansible-specific as such. In the past it was shipped as a standalone library and users had to install it from RPM or PyPI to be able to use the collection. This turned out to be a burden, especially for users running on enterprise distributions, so we decided to vendor the library inside the collection. The library development still happens separately on [GitHub](https://github.com/Apipie/apypie). Users now only have to install the `requests` library, which is easily available on any distribution.
```

Comment (melcorr) [inline _posts/2020-10-23-foreman-ansible-modules-v140-released.md]:
```suggestion
The biggest non-functional change in 1.4.0 is the vendoring of the `apypie` library. The library is responsible for the communication with the Foreman API and not Ansible-specific as such. In the past it was shipped as a standalone library and users had to install it from RPM or PyPI to be able to use the collection. This turned out to be a burden, especially for users running on enterprise distributions, so we decided to vendor the library inside the collection. The library development still happens separately on [GitHub](https://github.com/Apipie/apypie). Users now only have to install the `requests` library, which is easily available on any distribution.
```

File: _posts/2020-10-23-foreman-ansible-modules-v140-released.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+No new release without a new module, right? Well, this time you get two!
+
+[`job_invocation`](https://theforeman.github.io/foreman-ansible-modules/v1.4.0/plugins/job_invocation_module.html) allows you two run and schedule Remote Execution jobs from Ansible. You can even use Ansible to schedule an Ansible run that schedules an Ansible run…
```
Comment (melcorr) [inline _posts/2020-10-23-foreman-ansible-modules-v140-released.md]:
```suggestion
[`job_invocation`](https://theforeman.github.io/foreman-ansible-modules/v1.4.0/plugins/job_invocation_module.html) allows you to run and schedule Remote Execution jobs from Ansible. You can even use Ansible to schedule an Ansible run that schedules an Ansible run…
```

File: _posts/2020-10-23-foreman-ansible-modules-v140-released.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+[`smart_proxy`](https://theforeman.github.io/foreman-ansible-modules/v1.4.0/plugins/smart_proxy_module.html) allows to manage Smart Proxies, including setting the download policy and assigning lifecycle environments on Katello installations.
```
Comment (melcorr) [inline _posts/2020-10-23-foreman-ansible-modules-v140-released.md]:
```suggestion
[`smart_proxy`](https://theforeman.github.io/foreman-ansible-modules/v1.4.0/plugins/smart_proxy_module.html) allows you to manage Smart Proxies, including setting the download policy and assigning lifecycle environments on Katello installations.
```

**Review by melcorr — COMMENTED** [review]
Have I mentioned that you're really good at blogging @evgeni ?

**Comment by melcorr** [comment]
Few tiny things and otherwise it looks great. Exciting news.

**Pull request merged**